### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Remote Debugging-Debugger_Keyboard: When focus comes any of the list item in frames tab, visually user is unable to identify any change in the focus visibility.(there is no any color change or thick dotted border)

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3591]
GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.4.7; 

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 2.4.7

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till create new project-->Azure Spark/HDInsight-->Spark project with Samples(Scala) 
3)Tab till next button and press enter
4)Fill all the details correctly and Tab till finish button and press enter
5)Tab till <Project Name>-->src-->main-->scala-->sample-->SparkCore_wasbloTest and open it
6)Tab till Edit Configurations menu(select the icon in the upper-right corner)
7)Navigate to Advanced Configuration (Remote Debugging) section and Press the shortcut key to check Enable Spark remote debug
8)Fill Serure Shell (SSH) User Name and SSH password
9)Sent some breakpoints in SparkCore_wasbloTest code and tab till sparkjobdebug and press enter
10)Press cntrl+tab to navigate to Debug Remote Spark Job in Cluster section and verify user is able to identify  any change in the focus visibility when keyboard focus comes to list item in frames tab or not

**Actual Result:**
When focus comes any of the list item in frames tab, visually user is unable to identify any change in the focus visibility

**Expected Result:**
When focus comes any of the list item in frames tab, visually there should be a color change or a thick dotted border should be provided so that keyboard users will identify it.

**User Impact:**
Keyboard users will find difficult to understand where the actual focus is and they might think there is a focus loss in the frames tab

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
[Bug_3591_unable to identify the focus.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641665/Bug_3591_unable.to.identify.the.focus.pptx)

**Comment by t-rufang** [comment]
Note: When the focus stays at the following place and we press tab. Nothing changes.
![image](https://user-images.githubusercontent.com/32627233/65596933-8d636500-dfca-11e9-9b8a-6754f8d604da.png)

The debug windows is supported by IntelliJ rather than our tool.

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749694
